contracts/interfaces/IStealthVault.sol

```diff
@@ -9,20 +9,20 @@ interface IStealthVault {
   event PenaltyApplied(bytes32 _hash, address _caller, uint256 _penalty, address _reportedBy);
   event ValidatedHash(bytes32 _hash, address _caller, uint256 _penalty);
 
-  event StealthJobEnabled(address indexed _caller, address _job);
+  event StealthJobEnabled(address indexed _caller, address _contract);
 
-  event StealthJobsEnabled(address indexed _caller, address[] _jobs);
+  event StealthJobsEnabled(address indexed _caller, address[] _contracts);
 
-  event StealthJobDisabled(address indexed _caller, address _job);
+  event StealthJobDisabled(address indexed _caller, address _contract);
 
-  event StealthJobsDisabled(address indexed _caller, address[] _jobs);
+  event StealthJobsDisabled(address indexed _caller, address[] _contracts);
```

Comment:
Should we change this name events to the `contracts` naming ?

Reply:
oops, yes